libboardgame_gtp/CmdLine.cpp

```diff
@@ -66,7 +66,7 @@ string_view CmdLine::get_trimmed_line_after_elem(unsigned i) const
 {
     assert(i < m_elem.size());
     auto& e = m_elem[i];
-    auto begin = e.end();
+    auto begin = &*e.end();
```

Comment:
This change is to keep MSVC happy (it seems to not yet know how to automatically convert between iterator to string_view and char*)